frontend/tests/console/utils/DebugBundlePage.ts

```diff
@@ -0,0 +1,357 @@
+/** biome-ignore-all lint/performance/useTopLevelRegex: this is a test */
+import type { Download, Page } from '@playwright/test';
+import { expect } from '@playwright/test';
+
+/**
+ * Page Object Model for Debug Bundle pages
+ * Handles creation, progress monitoring, and bundle management
+ */
+export class DebugBundlePage {
+  readonly page: Page;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+
+  /**
+   * Navigation methods
+   */
+  async goto() {
+    await this.page.goto('/debug-bundle');
+    await expect(this.page.getByRole('heading', { name: /debug bundle/i })).toBeVisible();
```

Comment:
Shouldn't we `waitForURL` instead and check `.includes('/debug-bundle')` instead?